DFC.Digital/DFC.Digital.Web.Sitefinity.CourseModule.UnitTests/packages.config

```diff
@@ -10,10 +10,12 @@
   <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net472" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net472" />
   <package id="Moq" version="4.10.1" targetFramework="net472" />
+  <package id="MSTest.TestAdapter" version="1.3.2" targetFramework="net472" />
+  <package id="MSTest.TestFramework" version="1.3.2" targetFramework="net472" />
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net472" />
   <package id="RazorGenerator.Testing" version="2.4.7" targetFramework="net472" />
   <package id="ReflectionMagic" version="2.0.0" targetFramework="net472" />
-  <package id="StyleCop.Analyzers" version="1.0.2" targetFramework="net472" developmentDependency="true" />
+  <package id="StyleCop.Analyzers" version="1.1.118" targetFramework="net472" developmentDependency="true" />
```

Comment:
Ensure if this package needs to be updated, then they are consolidated across the solutions

Reply:
Package was reset back to 1.0.2 as it was before on this project